shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -323,6 +325,22 @@ private native void nativeDispatchPointerDataPacket(long nativePlatformViewId,
                                                       ByteBuffer buffer,
                                                       int position);
 
+  public void dispatchSemanticsAction(int id, @NonNull AccessibilityBridge.Action action) {
```

Comment:
Did you mean for this to be part of this PR?

Reply:
Yeah, it seemed like pointless logic within FlutterView, so I moved it to FlutterJNI where it seemed more legitimate to do the value translations.  But it shouldn't be a behavioral change, just a refactor.